src/Makefile

```diff
@@ -1861,6 +1830,7 @@ clean-internals: clean-unit-tests
 clean-unit-tests:
 	rm -f ${wrappers_syscheck_o}
 	rm -f ${wrappers_shared_o}
+	rm -f ${wrappers_common_o}
```

Comment:
this is adapting to how is the master branch: https://github.com/wazuh/wazuh/blob/ff962364251c1fa38468bc750c8431ced9254b84/src/Makefile#L1816